src/main/java/com/minecolonies/entity/ai/util/Structure.java

```diff
@@ -118,7 +118,7 @@ public Structure(World targetWorld, BlockPos buildingLocation, String schematicF
      * @param blockProgress     the block it should start with
      * @throws StructureException when there is an error loading the schematic file
      */
-    public Structure(World targetWorld, BlockPos buildingLocation, String schematicFileName, int rotation, Stage stageProgress, BlockPos blockProgress) throws StructureException
+    public Structure(final World targetWorld, final BlockPos buildingLocation, final String schematicFileName, final int rotation, final Stage stageProgress, final BlockPos blockProgress) throws StructureException
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 213 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)